cmd/clusterctl/internal/scheme/scheme.go

```diff
@@ -45,7 +44,6 @@ func init() {
 	_ = apiextensionsv1beta1.AddToScheme(Scheme)
 	_ = admissionregistration.AddToScheme(Scheme)
 	_ = admissionregistrationv1beta1.AddToScheme(Scheme)
-	_ = addonsv1.AddToScheme(Scheme)
```

Comment:
Not sure if that explains the unit test failure, but I think you have to add it again